basis/tools/continuations/continuations.factor

```diff
@@ -26,6 +26,47 @@ SYMBOL: break-hook
 
 \ break t "break?" set-word-prop
 
+SYMBOL: hit-counter
+
+: (hit-count-inc) ( -- )
+    hit-counter get dup
+    [  1 + ]
+    [ drop 1 ] if
+    hit-counter set ;
+
+: hit-count-reset ( -- )
+    0 hit-counter set ;
```

Comment:
I'm not sure about how this scoping will work, this sets it in the current namespace, and then the other words "get/set" it into the new namespace, so if your word does things like ``with-variable`` that binds a new namespace then maybe this won't work like you expect?

Reply:
Yes, you're right, I just checked it doesn't work unless you remain in the current namespace. Suggestions?

Reply:
On second thought, it's probably not worth pursuing. Just close the request. If I need it again will address it then.